IOPool/Input/test/testNoParentDictionary.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -e
+
+function die { echo $1: status $2 ;  exit $2; }
+
+SCRAM_TEST_NAME=TestIOPoolInputNoParentDictionary
+rm -rf $SCRAM_TEST_NAME
+mkdir $SCRAM_TEST_NAME
+cd $SCRAM_TEST_NAME
+
+# Create a new CMSSW dev area
+OLD_CMSSW_BASE=${CMSSW_BASE}
+scram -a $SCRAM_ARCH project $CMSSW_VERSION
+pushd $CMSSW_VERSION/src
+eval `scram run -sh`
+
+# Symlink any libraries and -edmplugincache *except* for the libraries containing the test dictionaries
+for F in ${OLD_CMSSW_BASE}/lib/${SCRAM_ARCH}/* ${OLD_CMSSW_BASE}/lib/${SCRAM_ARCH}/.edmplugincache; do
+    if [ -e ${F} ] && [[ "${F}" != *"DataFormatsTestObjectsParent1"* ]]; then
+        ln -s ${F} ${CMSSW_BASE}/lib/${SCRAM_ARCH}
+    fi
+done
```

Comment:
@makortel , during IB unit tests the `$OLD_CMSSW_BASE/lib/$SCRAM_ARCH` directory will empty so this unit test is not going to create any symlinks.

Reply:
I believe that is fine. What happens in PR tests, is `$OLD_CMSSW_BASE/lib/$SCRAM_ARCH` populated there? (I'd guess "yes")

This loop is for the case where the local developer area (`$OLD_CMSSW_BASE`) has code changes wrt. the release  (`$CMSSW_RELEASE_BASE`), which would be relevant for a developer running tests locally and (I believe) PR tests.